docs/content/how-to/disaster-recovery/backup-and-restore-oadp-1-5.md

```diff
@@ -0,0 +1,879 @@
+      need these PVCs because the VMs are going to be recreated as new VMs. We want tilter these PVCs out of the backup
+      to gain meaningful reduce in backup time and storage size. We'll filter these PVC using this label selector:
+
+        ```yaml
+        labelSelector:
+          matchExpressions:
+          - key: 'hypershift.openshift.io/is-kubevirt-rhcos'
+            operator: 'DoesNotExist'
+        ```
+
+    Once you apply the manifest, you can monitor the backup process in two places: the backup object status and the Velero logs. Please refer to the [Watching](#watching) section for more information.
+
+    The backup process is considered complete when the `status.phase` is `Completed`.
+
+
+## Restore
+
+For the restoration procedure, ensure that there are no Pods/PVCs running in the HostedControlPlane namespace to facilitate a successful restoration. If restoring the HostedCluster within the same Management cluster, delete the following objects:
+
+- HostedCluster
+- Nodepools
+- PVCs
+
+This will remove the Pods/PVCs, allowing for a proper restoration of the environment.
+
+!!! important
+
+    In the case of the Bare Metal provider (Agent), it's crucial to ensure that we don't delete the InfraEnv object. This object is mandatory for the new nodes that will be reprovisioned, as they need access to it in order to retrieve the Discovery ISO.
+
+The restoration object is provider agnostic. Just make sure you point to the right backupName to restore from
+
+Now let's take a look to the restoration manifest:
+
+```yaml
+---
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: hc-clusters-hosted-restore
+  namespace: openshift-adp
+spec:
+  backupName: hc-clusters-hosted-backup
+  restorePVs: true
+  existingResourcePolicy: update
+  excludedResources:
+  - nodes
+  - events
+  - events.events.k8s.io
+  - backups.velero.io
+  - restores.velero.io
+  - resticrepositories.velero.io
+```
+
+Here we have a couple of important fields
+
+```yaml
+restorePVs: true
+existingResourcePolicy: update
+```
+
+- `restorePvs` will initiate the recovery of pods with the included persistent volumes.
+- Setting `existingResourcePolicy` to `update` ensures that any existing objects are overwritten with the backup content. This may cause issues with objects containing immutable fields, which is why we delete them in a previous step. If this policy is not set, the Velero engine will skip the restoration of objects that already exist.
+
+You can monitor the restoration process by checking the restore status field and following the Velero logs mentioned in the [Watching](#watching) section.
+
+The restoration process is considered complete once the `status.phase` is `Completed`.
+
+!!! important
+
+    The restore may only be done on the same management cluster where the backup was created. Depending on the HostedCluster provider you are using, there are important topics to have in mind before the restoration.
```

Comment:
WDYT @celebdor, should we allow restoration in different MGMT cluster or keep in the same for now?